plugins/email_service/config/routes.rb

```diff
@@ -25,10 +24,15 @@
   resources :custom_verification_email_templates, only: [:index, :show, :new, :edit, :create, :destroy, :update]
 
   resources :multicloud_accounts, only: [:index, :show, :new, :edit, :create, :destroy]
-  resources :ec2_credentials, only: [:index, :create, :show] do 
-    member do 
+
+  resources :ec2_credentials, only: [:index, :create, :show] do
+    member do
       delete :destroy
     end
   end
 
+  # test helper methods with the hidden route
+  # get '/web' => 'web#index', as: :web
+  get '/web/test' => 'web#test', as: :test
```

Comment:
this is no a hidden rout if it is in the routes. Can you please test it using the used routes and not writing one just for testing?

Reply:
removed.